<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplikacja szyfrująca</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cipherButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px dotted; border-radius: 15px; color: #fff; background: #226fbe;</string>
    </property>
    <property name="text">
     <string>Szyfruj plik</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cipherText">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>531</width>
      <height>70</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Polish" country="Poland"/>
    </property>
    <property name="placeholderText">
     <string>Tekst do zaszyfrowania</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cipherResults">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>531</width>
      <height>70</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Rezultat szyfrowania</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>611</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="decipherButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>450</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px dotted; border-radius: 15px; color: #fff; background: #226fbe;</string>
    </property>
    <property name="text">
     <string>Deszyfruj plik</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopServerButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Zakończ serwer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runServerButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>89</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; border: 3px solid #fff; color: #000; background-color: #2ecc71;</string>
    </property>
    <property name="text">
     <string>Uruchom serwer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Połącz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Rozłącz</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="serverPort">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Server port</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgreen</string>
    </property>
    <property name="minimum">
     <number>1024</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="clientPort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Client port</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightgreen;</string>
    </property>
    <property name="minimum">
     <number>1024</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>5001</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="cipherModeCBC">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: brown;</string>
    </property>
    <property name="text">
     <string>CBC</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="cipherModeECB">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: brown;</string>
    </property>
    <property name="text">
     <string>ECB</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
